elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -159,18 +185,18 @@ public void inject(Object entity) {
     @Bean
     @ConditionalOnMissingBean
     @ConditionalOnProperty(name = "elide.aggregation-store.enabled", havingValue = "true")
-    public QueryEngine buildQueryEngine(EntityManagerFactory entityManagerFactory,
+    public QueryEngine buildQueryEngine(DataSource defaultDataSource,
```

Comment:
Does Spring provide the DataSource for us?

Reply:
yes, we are not creating DataSource anywhere.